cadastre/cadastre_import.py

```diff
@@ -459,12 +459,13 @@ def get_available_majic_files(self) -> tuple[dict, dict]:
                     if re.search(file_regex, os.path.split(file_sub_path)[1].upper()):
                         # Add file path to the list
                         file_path = os.path.join(root, file_sub_path)
-                        maj_list.append(file_path)
 
                         # ignore PDF/ODT files
                         if file_path.endswith((".PDF", ".pdf", ".ODT", ".odt")):
                             continue
 
+                        maj_list.append(file_path)
```

Comment:
I know it does not come from this PR, but I would use instead, and probably add more file types just in case :
```suggestion
                        if file_path.lower().endswith((".pdf", ".odt", ".doc", ".docx", ".txt")):
                            continue

                        maj_list.append(file_path)
```

Reply:
i wouldnt add `.txt` since the files i generate from the original files have that extension ;)

but yeah i agree doc/docx belong there, and i should have thought earlier about using `.lower()`.. thanks for that suggestion :)